Plugins/NativeServerPlugin/inc/conductor.h

```diff
@@ -142,26 +139,31 @@ class Conductor : public webrtc::PeerConnectionObserver,
 
 	void OnFailure(const std::string& error) override;
 
+private:
```

Comment:
nit: private should fall below protected in order (that way a reader can understand the api space from most available [public and callable from everywhere] to least available [private and only internal])

Reply:
Yes, it's indeed a bug from my refactoring.